docs/variants/pc_engines/initial-deployment.md

```diff
@@ -0,0 +1,85 @@
+# Initial deployment
+
+This document describes the process of updating
+[PC Engines coreboot firmware](https://pcengines.github.io/) to Dasharo firmware
+based on EDK II.
+
+## OS compatibility
+
+Switching from BIOS to UEFI-based firmware might cause compatibility issues if
+you already had an operating system installed on your platform. Some OSs can
+handle the switch without any issues, while others may need to be reinstalled.
+We have tested the update on several operating systems. The results are
+presented in the table below:
+
+| OS | Can be booted from UEFI  |
+|----|------------------------- |
+| Ubuntu | :heavy_check_mark:   |
+| OPNSense | :heavy_check_mark: |
+| pfSense | :x:                 |
+| OpenWrt | Depends[^1]         |
+
+Initial flashing of Dasharo firmware can be done from Linux using flashrom
+with the internal programmer.
+
+## Deploy using Dasharo Tools Suite
+
+For simplicity we recommend using [Dasharo Tools
+Suite](../../dasharo-tools-suite/documentation.md#dasharo-zero-touch-initial-deployment)
+to omit all compilation steps and deploy the Dasharo firmware seamlessly.
+
+## Build flashrom
+
+Please follow generic guide for [flashrom
+building](https://www.flashrom.org/dev_guide/building_from_source.html).
+
+Or install it from the OS' package manager (minim supported version is v1.0).
+
+## Reading flash contents
+
+Always prepare a backup of the current firmware image. If you are using DTS,
+the backup will be made automatically with [HCL
+report](../../dasharo-tools-suite/documentation.md#hcl-report). When deploying
+manually, to read from the flash and save it to a file (`dump.rom`), execute
+the following command:
+
+```bash
+flashrom -p internal -r dump.rom
+```
+
+## Flashing Dasharo
```

Comment:
I think we are missing some steps, e.g. entering DES credentials like we explained here: https://docs.dasharo.com/variants/novacustom_nv4x_adl/heads/#switching-from-dasharo-uefi-to-heads

Reply:
The manual installation steps here don't need them, the binaries are downloaded manually by the user in this case.

What we are missing, are initial deployment steps using DTS.

Reply:
https://github.com/Dasharo/docs/pull/794/commits/74587a95249252cd7b8aa33a6bf4c31e423c2b44